src/corehost/cli/fxr/fx_muxer.cpp

```diff
@@ -843,6 +957,30 @@ int muxer_usage(bool is_sdk_present)
     return StatusCode::InvalidArgFailure;
 }
 
+int versions_runtime_info(pal::string_t own_dir)
+{
+    std::vector<pal::string_t> fx_versions;
+    get_all_fx_versions(host_mode_t::muxer, own_dir, _X("Microsoft.NETCore.App"), &fx_versions);
```

Comment:
What do you think about discovering ALL the frameworks under the `shared` directory instead of just `Microsoft.NETCore.App`?  I know we haven't used the ability to make more shared frameworks yet, but if we do this code won't find them.

Reply:
Perhaps overkill, but I did add that. It will now list the framework name first, then version, then directory